src/mantarray_waveform_analysis/pipelines.py

```diff
@@ -200,13 +205,13 @@ def get_magnetic_data_metrics(
             ],
         ],
     ]:
-        """Calculate data metrics on noise filtered magnetic data."""
+        """Calculate data metrics on force data."""
```

Comment:
sorry if this wasn't clear earlier---eventually we may need to display to the users metrics calculated either from the magnetic data, displacement data, or force data.  So we should keep this present method like it was before, but we ened to add `get_displacement_data_metrics` and `get_force_data_metrics`